src/components/ManageGallery/OrganizeCollection/Sidebar/SidebarList.tsx

```diff
@@ -203,3 +282,17 @@ const Selection = styled.div`
   display: flex;
   grid-gap: ${SIDEBAR_ICON_GAP}px;
 `;
+
+const ShowHideContainer = styled.div`
+  display: flex;
+  align-items: center;
+  margin-left: auto;
+`;
+
+const StyledTooltip = styled(Tooltip)<{ visible: boolean }>`
+  opacity: ${({ visible }) => (visible ? 1 : 0)};
+  width: 50%;
+  right: 0;
+  top: 30px;
+  z-index: 1;
+`;
```

Comment:
could we make the tooltip a bit wider? it's 130px in figma

<img width="325" alt="image" src="https://user-images.githubusercontent.com/12162433/201436743-d78e7992-2def-477f-a14c-6e4331466244.png">

wondering if we could position it this way as well:

<img width="130" alt="image" src="https://user-images.githubusercontent.com/12162433/201436786-0ffefad6-d92f-4fce-a9dd-fc71a6d69d8e.png">

Reply:
ooh yes!

and re: positioning, i think `<Tooltip />` would need to be reworked to use a react portal so that it can render outside of the sidebar! i was worried it might affect other places it's used in unexpected ways, but i can def add that—thoughts? i think it does make sense to do that in general, probably also for `<Dropdown />`

an easier/more reliable way to handle it versus rolling it from scratch would be using a library (i.e., https://floating-ui.com/docs/floatingportal) that handles positioning, focus, etc., out of the box

Reply:
ahh wow forgot about the sidebar, and great point about using a portal - that sounds great!

i don't think this is blocking, so i'll create a separate ticket with the floating portal suggestion as a fast-follow :) no need to handle in this PR